e2e/components/cookies-banner/cookie-banner-accepted.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { expect, test } from '@playwright/test'
```

Comment:
Directory name should be `cookie-banner` here (I did think it was cookies banner before though...)

Also in #22 I left out the component name from the filename, do you think we need it?